test/parallel_api/ranges/std_ranges_test.h

```diff
@@ -171,6 +176,14 @@ constexpr int trivial_size{0};
 template<typename>
 constexpr int calc_res_size(int n, int) { return n; }
 
+// If in1 range is empty, then out range is empty
+template<typename>
+int out_size_with_empty_in1(int) { return 0; };
+
+// If in2 range is empty, then out range is empty
+template<typename>
+int out_size_with_empty_in2(int) { return 0; };
+
```

Comment:
Add a comment that these functions can be specialized in the actual tests.

Reply:
Done see: [0015ba7](https://github.com/uxlfoundation/oneDPL/pull/2320/commits/0015ba7c4a3dcf7abcee89e6a057c1655804b57a).